.github/workflows/upload-release-to-plugin-repo.yml

```diff
@@ -130,8 +145,8 @@ jobs:
             - name: Replace the stable tag placeholder with the existing stable tag on the SVN repository
               env:
                   STABLE_TAG_PLACEHOLDER: 'Stable tag: V\.V\.V'
-                  STABLE_TAG: 'Stable tag: ${{ steps.get_previous_stable_version.outputs.stable_version }}'
-              run: sed -i "s/${STABLE_TAG_PLACEHOLDER}/${STABLE_TAG}/g" ./trunk/readme.txt
+              run: |
```

Comment:
Before, `STABLE_TAG` was based on the value of `steps.get_previous_stable_version.outputs.stable_version`, which for me looked a bit confusing, given the `get_previous_stable_version` step -- we actually want the current latest version, and not the previous, and we can get that from the `VERSION` env variable, defined as an `env` variable for the job and passed as part of the payload for the `release` event.